lib/set_order_command.rb

```diff
@@ -0,0 +1,35 @@
+require 'order'
+
+class SetOrderCommand
+  attr_reader :response
+
+  def initialize(lunch, data)
+    @lunch = lunch
+    @user_id = data[:user_id]
+    @user_name = data[:user_name]
+  end
+
+  def response?
+    response
+  end
+
+  def run()
+    order = Order.last(:user_id => @user_id)
+
+    if order
+      order.lunch = @lunch
+      order.save
+    else
+      order = Order.new
+      order.attributes = {
```

Comment:
Can this be set as part of the construct?
`Ordner.new({user_name: @user_name, ...})`?